engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -218,6 +245,10 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
     s"$workflowId-BackendJobExecutionActor-${jobDescriptor.key.tag}"
   }
 
+  private def buildCacheHitCopyingActorName(jobDescriptor: BackendJobDescriptor) = {
+    s"$workflowId-BackendCacheHitCopyingActor-${jobDescriptor.key.tag}"
```

Comment:
ToL: We can come back and clean this up later, but I wonder how many times `$workflowId` is already in the actor path